### DISCUSSION TITLE
is it snell support SNI?

### DISCUSSION THREAD

**Discussion by malikshi** [icpz/open-snell#6] [Q&A]
hi, i am new to open-snell or snell server, is it open-snell support `SNI` or `servername` and `allowinsecure=true` ?
i saw your commit in clash repository so i want to try open-snell

**Comment by icpz** [comment]
No, this is the open source implementation of [surge-networks/snell](https://github.com/surge-networks/snell). So most of the features are identical to the original project.

The original project does not support SNI stuffs since the `tls` mode in `snell` is not the fully functional `tls`, please consider it as a fake `tls` mode only for obfuscating.

**Reply by malikshi** [reply]
thank you for the answers, i gonna check & test open-snell hope good as alternative proxy
